services/app-api/handlers/forms/post/generateQuarterFormsCron.js

```diff
@@ -0,0 +1,380 @@
+import handler from "../../../libs/handler-lib";
+import dynamoDb from "../../../libs/dynamodb-lib";
+import {
+  getFormDescriptions,
+  getQuestionsByYear,
+  getStatesList,
+  findExistingStateForms,
+  fetchOrCreateQuestions,
+  getAnswersSet,
+} from "../../shared/sharedFunctions";
+
+/**
+ * Generates initial form data and statuses for all states given a year and quarter
+ * NOTE: This function is triggered on a schedule by Cron
+ */
+
+export const main = handler(async (event, context) => {
```

Comment:
I think a lot of this and the generateQuarters file share code, is it possible to move some of the logic to a shared file after permissions are vetted?